redisgraph/execution_plan.py

```diff
@@ -0,0 +1,35 @@
+class ExecutionPlan:
+    def __init__(self, plan):
+        self.plan = plan
+
+    def __str__(self) -> str:
+        return "\n".join(self.plan)
+
+    def operation_tree(self):
+        stack = []
+        level = 0
+        current = None
+        i = 0
+        while i < len(self.plan):
+            op = self.plan[i]
+            op_level = op.count("    ")
+            if op_level == level:
+                current = {"op": op.replace("    ", "")}
+                i += 1
+            elif op_level == level + 1:
+                child = {"op": op.replace("    ", "")}
```

Comment:
I think `op` should be better structured, probably should be a class with at least a `name` and `arguments` attributes